### ISSUE TITLE
[3.0.0-beta7] chat and public note do not open on android

### ISSUE THREAD

**Issue by himpierre** [bigbluebutton/bigbluebutton#21930]
**Describe the bug**

Chat and public note do not open on android.

**To Reproduce**
1. Start meeting
2. Enter meeting
3. Try to open chat or public note
4. Tapping does not have any effect

**Expected behavior**
Chat window opens. Same goes for public note.

**Actual behavior**
Nothing happens.

**Screenshots**

**BBB version:**
3.0.0-beta7

**Smartphone (please complete the following information):**
 - Device: Pixel 6
 - OS: Android 15
 - Browser Chromium
 - Version 131

**Comment by Ithanil** [comment]
Can (kind of) confirm this, but it only happens if the screen is too narrow, which is typically the case for phones held upright. If you rotate the phone and display into a horizontal position, chat & shared notes should show.

**Comment by himpierre** [comment]
Yes, you're right! Rotating to horizontal fixes the problem. I guess that's something the dev's can work with. :)

cheers!

Am 20. Dezember 2024 09:21:38 MEZ schrieb Jan Kessler ***@***.***>:
>Can (kind of) confirm this, but it only happens if the screen is too narrow, which is typically the case for phones held upright. If you rotate the phone and display into a horizontal position, chat & shared notes should show.
>

**Comment by Ithanil** [comment]
@himpierre Yeah, let's hope so.

And it's definitely a regression from BBB 2.7, where opening the chat would simply overlay the chat over the sidebar on a narrow screen.

**Comment by shariquealavi** [comment]
I upgraded our instance to this beta7 by mistake ... and this bug is killing it for us ... 😞
